src/desktop/apps/article/routes.js

```diff
@@ -132,6 +134,8 @@ export async function index(req, res, next) {
     const showTooltips = !isMobile && !isTablet
     const renderTime = getCurrentUnixTimestamp()
 
+    res.locals.sd.RESPONSIVE_CSS = createMediaStyle()
```

Comment:
One small optimization would be to invoke the function in the top-level scope once and re-use the result for each request.

Reply:
Going to merge so it is up during my QA meeting, will return to this suggestion!